src/services/Login.ts

```diff
@@ -83,12 +83,24 @@ class OpenshiftLogin implements LoginStrategy<any> {
   }
 
   public async perform(_request: NullDispatch): Promise<LoginResult> {
-    const session = (await API.checkOpenshiftAuth(window.location.hash.substring(1))).data;
+    try {
+      const session = (await API.checkOpenshiftAuth(window.location.hash.substring(1))).data;
 
-    return {
-      status: AuthResult.SUCCESS,
-      session: session
-    };
+      // Remove the access_token information from the URL so that if we have a reload on LoginThunkActions.checkCredentials
+      // it will take us back to the OpenShift Login page. We don't want to redirect now, but affect the prepare check above.
+      history.pushState('', document.title, window.location.pathname + window.location.search);
```

Comment:
In this case, I think `replaceState` is better than `pushState`

Reply:
Done